pkg/config/storage.go

```diff
@@ -0,0 +1,76 @@
+package config
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+const driverUnset = ""
+const driverNone = "none"
+const driverLVMS = "lvms"
+
+const csiComponentNone = "none"
+const csiComponentSnapshot = "snapshot-controller"
+const csiComponentSnapshotWebhook = "snapshot-webhook"
+
+// Storage represents a subfield of the microshift config data structure. Its purpose to provide a user
+// facing interface to control whether MicroShift should deploy LVMS on startup.
+type Storage struct {
+	// One of ["lvms", "none", ""]
+	Driver        string         `json:"driver,omitempty"`
```

Comment:
Given that we have a closed set of values, should this be a different type to express it more clearly? Like the enums in k8s apis, for example.